syft/serde/msgpack/serde.py

```diff
@@ -446,6 +454,13 @@ def _simplify(worker: AbstractWorker, obj: object, **kwargs) -> object:
         return obj
 
 
+def _detail_field(typeCode, val):
```

Comment:
Maybe write `str(2^64)` instead of "18446744073709551616" so that it's clearer what is means

Reply:
Oh yeah, sure. I will change it